README.md

```diff
@@ -21,6 +21,14 @@ git clone git@github.com:lagom/online-auction-java.git
 cd online-auction-java
 sbt runAll
 ```
+## Importing into IDEs
+
+### Intellij IDEA
+You will need to install following plugins into IDEA by going to `File -> Settings -> Plugins`
+* **[SBT](http://github.com/orfjackal/idea-sbt-plugin)** : SBT build tool plugin.
```

Comment:
Hi @lakhina I think the sbt plugin for IDEA is somewhat discontinued in favour of the scala plugin (https://plugins.jetbrains.com/idea/plugin/1347-scala vs https://plugins.jetbrains.com/idea/plugin/5007-sbt). 
BTW, @rstento is reworking the `integrating with IDE`s bits (https://github.com/lagom/lagom/pull/516) in Lagom's docs.
Also, it'd be great if instructions included Eclipse. We're trying to keep parity with at least IntelliJ and Eclipse.

Keep up the good work! ;-)